src/components/app/app.js

```diff
@@ -1,6 +1,21 @@
 import angular from "angular";
 import template from "./app.html";
 
-export const appModule = angular.module("appModule", []).component("app", {
-  template
-}).name;
+import ngMaterial from "angular-material";
+import ngAnimate from "angular-animate";
+import ngAria from "angular-aria";
+
+import { transactionDialogModule } from "../transaction-dialog/transaction-dialog";
+
+export class AppController {
+  handleAddTransactionClick(event) {
+    this.transactionDialogOpen = { event: event };
```

Comment:
please pass event as separate prop -- it is really confusing to see prop `is-open` on dialog and you pass event wrapper into an object there